netci.groovy

```diff
@@ -506,7 +506,15 @@ def static addTriggers(def job, def branch, def isPR, def architecture, def os,
                 // GCStress=C is currently not supported on OS X
                 if (os != 'CentOS7.1' && os != 'OSX' && !(os in bidailyCrossList)) {
                     assert (os == 'Windows_NT') || (os in Constants.crossList)
-                    Utilities.addPeriodicTrigger(job, '@weekly')
+                    if (architecture == 'arm64') {
+                        assert (os == 'Windows_NT')
```

Comment:
Well. This is intentional. arm64 is enabled only for Windows not cross OSes, yet.